buildbot/dependency.py

```diff
@@ -50,29 +50,32 @@ def do_dependency(args):
     ocl_header_dir = os.path.join(args.obj_dir, "OpenCL-Headers")
     if not os.path.isdir(ocl_header_dir):
         clone_cmd = ["git", "clone", "https://github.com/KhronosGroup/OpenCL-Headers",
-                     "OpenCL-Headers", "-b", "v2020.06.16"]
+                     "OpenCL-Headers", "-b", "master"]
         subprocess.check_call(clone_cmd, cwd=args.obj_dir)
     else:
         fetch_cmd = ["git", "pull", "--ff", "--ff-only", "origin"]
         subprocess.check_call(fetch_cmd, cwd=ocl_header_dir)
 
-    # Workaround to unblock CI until KhronosGroup/OpenCL-ICD-Loader/pull/124
-    # is submitted
-    checkout_cmd = ["git", "checkout", "d1b936b72b9610626ecab8a991cec18348fba047"]
+    # Checkout fixed version to avoid unrelated CI failures
```

Comment:
```suggestion
    # Checkout fixed version to avoid unexpected issues coming from upstream
    # Specific version can be uplifted as soon as such need arise
```